### DISCUSSION TITLE
Multi user support?

### DISCUSSION THREAD

**Discussion by 5hay** [immich-app/immich#56] [Q&A] [answered]
Hi,

I have not tried to install it yet, just trying to evaluate if this project is suited for my use case. Is there multi user support built in? I can see the API endpoint to create an user in the README, but can I create multiple users with their own photo library or is it just for creating an admin account? If not, are there plans to support this use case in the future or is this designed as a single user solution?

Thank you.

**Comment by alextran1502** [comment, marked as answer]
Hey, 

You can create multiple users to use the same server setup with the endpoint in readme file.

Each user will have their own storage folder

**Reply by 5hay** [reply]
Sounds great, definitely gonna give this a try then. Thank you very much.

I think that's a big selling point, it should be explained in the README.

**Reply by alextran1502** [reply]
Thanks for the suggestion, I will put that in as a feature :)
